v1.2-client.yml

```diff
@@ -1548,6 +1693,13 @@ definitions:
         format: datetime
         x-nullable: true
 
+  Credentials:
+    type: object
+    properties:
+      district_username:
+        type: string
+        x-nullable: true
+
```

Comment:
Remove x-nullable:true here and leave it to the higher level object like we do for GoogleApps